build.sbt

```diff
@@ -123,36 +123,6 @@ lazy val codegenJavaCompilationTest = project
     name := "akkaserverless-codegen-java-compilation-tests",
     Compile / javacOptions ++= Seq("-encoding", "UTF-8", "-source", "11", "-target", "11"))
 
-lazy val samples = project
-  .in(file("samples"))
-  .aggregate( // samples relying on codegen must use Maven
-    // FIXME consider including this sample again?
-    // `java-eventing-shopping-cart`,
-    // `valueentity-counter`
-  )
```

Comment:
The samples that can be compiled with sbt are already included directly. All the others are build with maven only by CI. We can remove this root project.

Reply:
AFAICT the `valueentity-counter` *can* be compiled from sbt when we put the generated code in `valueentity-generated`. That would make it easier to test changes to the runtime that don't affect the codegen (since you could rebuild and run from sbt instead of doing the 'publishLocal' dance). Whether that's worth it is a judgement call, of course.

Reply:
I don't think it's worth keeping it in the sbt build. `publishM2` works fine and then development in the sample projects with their real toolchain.

Note that when developing the version typically don't change (you keep the same sbt session alive) so publishM2 produce the same dev version.